plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -2973,7 +2974,7 @@ public Domain getDomain(final Connect conn, final String vmName) throws LibvirtE
     }
 
     private String executeBashScript(final String script) {
-        final Script command = new Script("/bin/bash", _timeout, s_logger);
+        final Script command = new Script("/bin/bash", _timeout.getMillis(), s_logger);
```

Comment:
Scripts is an external package, hope they will upgrade to Duration after Java 1.8.